--- icons/Makefile.orig	2020-12-10 16:07:28.080018600 -0500
+++ icons/Makefile	2020-12-10 16:11:56.029344500 -0500
@@ -840,8 +840,8 @@
 .PRECIOUS: Makefile
 
 
-install-fake-index-theme: fake-index-theme
-	install -D -m 0644 fake-index-theme $(datadir)/icons/hicolor/index.theme
+install-fake-index-theme: $(top_srcdir)/icons/fake-index-theme
+	install -D -m 0644 $(top_srcdir)/icons/fake-index-theme $(datadir)/icons/hicolor/index.theme
 
 install-data-hook: update-icon-cache $(fake_index_theme_target)
 uninstall-hook: update-icon-cache
